docs/checks.md

```diff
@@ -35,7 +35,7 @@ This check determines if the default and release branches are protected with Git
 
 This check tries to determine if the project runs tests before pull requests are merged.
 Running tests helps developers catch mistakes early on. A low score is considered 'Low' risk.
-The check works by looking for a set of well-known CI-system names in GitHub `CheckRuns` and `Statuses` among the recent commits (~30). A CI-system is considered well-known if its name contains any of the following: appveyor, buildkite, circleci, e2e, github-actions, jenkins, mergeable, test, travis-ci. The check succeeds if at least 75% of successful pull requests have at least one successful check associated with them. 
+The check works by looking for a set of well-known CI-system names in GitHub `CheckRuns` and `Statuses` among the recent commits (~30). A CI-system is considered well-known if its name contains any of the following: appveyor, buildkite, circleci, e2e, github-actions, jenkins, mergeable, test, travis-ci. The check succeeds if at least 75% of successful pull requests have at least one successful check associated with them. A project may meet this criterion yet have a failing scorecard report; there are many ways to implement this criterion and it's especially difficult for an automated tool (like scorecard) to detect them all. 
```

Comment:
Suggestion `A project may meet this criterion yet have a failing scorecard report; there are many ways to implement this criterion, and it's challenging for an automated tool (like scorecard) to detect them all. `

Reply:
1. `The check succeeds if at least 75% of successful pull requests have at least one successful check associated with them.` please remove this sentence because our checks no longer fail or pass.  The score is calculated as number_reviews/reviewed_looked_at; and reivews_looked_at is currently 30.
2. The check only has false negatives and many can be removed by adding more tool names: please let's also drop a line saying that "if you think your system was not detected, open an issue at https://github.com/ossf/scorecard/issues/new/choose"

Reply:
Will do.